relay-general/src/protocol/event.rs

```diff
@@ -468,6 +468,10 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub spans: Annotated<Array<Span>>,
 
+    /// Measurements
+    #[metastructure(skip_serialization = "empty_deep")]
+    pub measurements: Annotated<Measurements>,
```

Comment:
@fpacifici the measurements interface.

Reply:
Please add `omit_from_schema` unless this is part of error events as well:

```
#[metastructure(omit_from_schema, ...)]
```

Reply:
@untitaker done in https://github.com/getsentry/relay/pull/724/commits/79bba11cbd52b662cd5503e72f3f89f2f2ef9c0f